File: app/src/main/java/com/dhsdevelopments/potato/clientapi/notifications/PotatoNotification.java
package com.dhsdevelopments.potato.clientapi.notifications;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class PotatoNotification implements Serializable
{
    @SerializedName( "type" )
    public String type;


    @Override
    public String toString() {
        return "PotatoNotification[" +
                       "type='" + type + '\'' +
                       ']';
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/notifications/PotatoNotification.kt
package com.dhsdevelopments.potato.clientapi.notifications

import com.google.gson.annotations.SerializedName

import java.io.Serializable

open class PotatoNotification : Serializable {
    @SerializedName("type")
    lateinit var type: String


    override fun toString(): String {
        return "PotatoNotification[type='$type']"
    }
}
